src/sb_gazebo/urdf/jfrost.gazebo

```diff
@@ -162,8 +162,8 @@
             <!-- Set these parameters depending on where you want to gps reference to be -->
             <!-- NOTE: THESE SHOULD MATCH THE COMPASS_SIM PLUGIN - SEE BELOW-->
             <!-- EDC (Very Roughly) -->
-            <referenceLongitude>49.262479</referenceLongitude>
```

Comment:
`<!-- NOTE: THESE SHOULD MATCH THE COMPASS_SIM PLUGIN - SEE BELOW-->`
If you're changing these up change the ones below as well?